### PR TITLE
feat(ui): Add Simplified Chinese localization support

### PR DESCRIPTION
**Pull request by shizn**
## Description

Add Simplified Chinese localization support.

## Related Issues

N/A

## Checklist

Before you create this PR confirm that it meets all requirements listed below by checking the relevant checkboxes (`[x]`).
This will ensure a smooth and quick review process. Updating the `pubspec.yaml` and changelogs is not required.

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [ ] My PR includes unit or integration tests for *all* changed/updated/fixed behaviors (See [Contributor Guide]).
- [ ] All existing and new tests are passing.
- [ ] I updated/added relevant documentation (doc comments with `///`).
- [ ] The analyzer (`melos run analyze`) does not report any problems on my PR.
- [x] I read and followed the [Flutter Style Guide].
- [x] I signed the [CLA].
- [x] I am willing to follow-up on review comments in a timely manner.

## Breaking Change

Does your PR require plugin users to manually update their apps to accommodate your change?

- [ ] Yes, this is a breaking change.
- [x] No, this is *not* a breaking change.

[issue database]: https://github.com/flutter/flutter/issues
[Contributor Guide]: https://github.com/FirebaseExtended/flutterfire/blob/master/CONTRIBUTING.md
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[pub versioning philosophy]: https://dart.dev/tools/pub/versioning
[CLA]: https://cla.developers.google.com/

### CODE REVIEW

File: packages/flutterfire_ui/lib/src/i10n/default_localizations.dart [hunk 1]
```diff
@@ -120,6 +121,7 @@ const localizations = <String, FlutterFireUILocalizationLabels>{
   'nl': NlLocalizations(),
   'id': IdLocalizations(),
   'de': DeLocalizations(),
+  'zh-CN': ZhCnLocalizations(),
```
Comment (russellwheatley) [inline packages/flutterfire_ui/lib/src/i10n/default_localizations.dart:124]: Do you mind changing this to just `zh`? It keeps it consistent with the rest of the language codes. This also means changing the class name to `ZhLocalizations`.

File: packages/flutterfire_ui/lib/src/i10n/lang/zh-CN.dart [hunk 2]
```diff
@@ -0,0 +1,263 @@
...
+  @override
+  final String latitudeLabel;
+  @override
+  final String longitudeLabel;
+  @override
+  final String geopointLabel;
+  @override
+  final String referenceLabel;
+
+  const EnLocalizations({
```
Comment (russellwheatley) [inline packages/flutterfire_ui/lib/src/i10n/lang/zh-CN.dart:171]: Change to `ZhLocalizations`.

**Review by russellwheatley — CHANGES_REQUESTED** [review]
Hey @shizn, thanks for the PR! Do you mind having a fellow Chinese speaking person proof read the Chinese language and put an approval/request changes on this PR, please? 😄

File: packages/flutterfire_ui/lib/src/i10n/default_localizations.dart (hunk 1 above)
Comment (shizn) [reply packages/flutterfire_ui/lib/src/i10n/default_localizations.dart]: Updated. You might encounter this issue again if someone contributes traditional Chinese in the future. :)

File: packages/flutterfire_ui/lib/src/i10n/lang/zh-CN.dart (hunk 2 above)
Comment (shizn) [reply packages/flutterfire_ui/lib/src/i10n/lang/zh-CN.dart]: Resovled.

**Comment by shizn** [comment]
Could you help take a look and put an approval/request changes on this PR? @yalunqin @cyanglaz. Found you in the top contributor list of this repo, and I presume you can speak Chinese.

**Comment by russellwheatley** [comment]
Hey @shizn, if you have a fellow software developer colleague/friend who can help, that works for me as well 😄. Thanks.

**Comment by inkpool** [comment]
LGTM. All Chinese is fine.

**Comment by russellwheatley** [comment]
Hey @shizn, you need to fix the CI issues. Thanks.

**Comment by shizn** [comment]
Hope I fixed all issues this time. 🤞 Need your help to approve the CI job and verify.

**Comment by russellwheatley** [comment]
@shizn  CI is still failing I'm afraid. You need to click the details button next to the CI test failing and update what needs changing, please.

**Comment by shizn** [comment]
Hey, can someone give a hint what was wrong? The latest CI job was cancelled, and I cannot find the error message. Manually checked these changes locally by comparing with other localization files, it seems good. I would like to know what the issue from the CI output was.

**Comment by russellwheatley** [comment]
@shizn, you should be able to see what is wrong now.

analyze issue:
```bash
info - lib/src/i10n/lang/zh.dart:221:29 - Only use double quotes for strings containing single quotes. - prefer_single_quotes
```

format this file `flutterfire_ui/lib/src/i10n/lang/zh.dart`

Thanks 🙏

**Comment by shizn** [comment]
Got it. Working on the fix. Thanks for the logs!

**Review by russellwheatley — CHANGES_REQUESTED** [review]
format the file as per the comment, thanks 😄

**Pull request closed without merging**